ui/server/src/main/scala/pl/touk/nussknacker/ui/api/ManagementResources.scala

```diff
@@ -150,11 +161,13 @@ class ManagementResources(processCounter: ProcessCounter,
       path("adminProcessManagement" / "deploy" / Segment ) { processName =>
         (post & processId(processName) & parameters('savepointPath)) { (processId, savepointPath) =>
           canDeploy(processId) {
-            withComment { comment =>
+            withDeploymentComment { deploymentComment =>
               complete {
-                processService
-                  .deployProcess(processId, Some(savepointPath), comment)
+                val eventualResponse = processService
```

Comment:
Yo don't need to create val. Please use calling method.

Reply:
changed it, thx